### ISSUE TITLE
[Mangadex] displays [END] on chapter 0 by default 

### ISSUE THREAD

**Issue by ZaneHannanAU** [inorichi/tachiyomi-extensions#380]
![example](https://cdn.discordapp.com/attachments/403909858287550467/464020917329199104/Screenshot_20180704-204830.png)

[Page](https://mangadex.org/manga/22987/)

Basically should only add the [END] or completed stati on the actual manga getting an end and being marked as complete.

Of course this isn't possible everywhere but probably most places.

Can be fixed on server end as said [on the mangadex forums](https://mangadex.org/thread/14311/tachiyomi-mangadex-often-displays-end-on-chapter-0) and here.

Basically changing the way mangadex defaults its `last_chapter` eg to `null` or `-1` are both functional; as well as checking the status is "complete" as set in source.

Thank you.
